modules/mono/editor/GodotTools/GodotTools/Export/ExportPlugin.cs

```diff
@@ -108,15 +110,19 @@ private void _ExportBeginImpl(string[] features, bool isDebug, string path, long
             {
                 archs.Add("x86_64");
             }
-            else if (features.Contains("x86_32"))
+            if (features.Contains("x86_32"))
```

Comment:
Why remove the `else`? Is android similar to mac universal exports?

Reply:
Yes, apks can target multiple architectures as well, and thus may need to contain the build for all. Unlike `universal` there is no dedicated feature tag, the export just has multiple tags.

(This reminds me that I wanted to check how exactly `universal` works and if the universal case or the distinct can be removed)